packages/e2e-tests/specs/editor/various/preview.test.js

```diff
@@ -144,6 +146,7 @@ describe( 'Preview', () => {
 		// Pressing preview without changes should bring same preview window to
 		// front and reload, but should not show interstitial.
 		await editorPage.bringToFront();
+		await editorPage.click( '.editor-post-preview__button-toggle' );
```

Comment:
Something I don't understand here.

We click on "editor-post-preview__button-toggle" here and then when calling `waitForPreviewNavigation`. Do we need to click the button twice?

Reply:
When navigating back from the preview to the editor, the preview options dropdown is still open, so when `waitForPreviewNavigation` clicks on it it closes instead of opening. The alternative to doing it this way would be _not_ opening the dropdown inside `waitForPreviewNavigation` and instead opening it beforehand if it's not already open. Perhaps it will be less confusing that way.

Reply:
`waitForPreviewNavigation` could also check if the dropdown is already open and only click the toggle when it needs to.

Reply:
I thought about that at first, but is it good practice to have conditionals in tests? It always feels a bit wrong to me 😅 and it would probably make the test slower because we'd have to wait to see if the selector was found.
Happy to do it if everyone agrees though!

Reply:
I suppose it makes that part of the test no longer deterministic. Maybe the easiest thing would be to break `waitForPreviewNavigation` into two functions in that case.

Reply:
I don't mind either conditions or breaking into two functions. Other than that, the changes here look good to me.

Reply:
Done!